mobile/src/bundles/employer/screens/candidates-filter.tsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialIcons';
+
+import {
+    AutocompleteMultiSelector,
+    Button,
+    CheckboxGroup,
+    Divider,
+    FormField,
+    Pressable,
+    ScrollView,
+    Selector,
+    Switch,
+    Text,
+    View,
+} from '~/bundles/common/components/components';
+import { Color, IconName } from '~/bundles/common/enums/enums';
+import { TextCategory } from '~/bundles/common/enums/styles/styles';
+import { useAppForm, useCallback } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+import {
+    CountryList,
+    EmploymentType,
+    JobTitle,
+} from '~/bundles/employer/enums/enums';
+
+import {
+    BSA_BADGES,
+    BSA_CHARACTERISTICS,
+    BSA_PROJECT,
+    DEFAULT_VALUES,
+    ENGLISH_LEVEL,
+    YEARS_EXPERIENCE,
+} from './constants/constants';
+
+const jobTitleOptions = Object.values(JobTitle);
```

Comment:
When my [PR](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/409) will be merged, you will need to change it because `AutocompleteMultiSelector`  accepts an array of objects in the format `{label: string, value:string}`

Or you can just cherry pick commits regarding autocomplete to avoid conflicts in the future

Reply:
You didnt got any reviews there, if I will cherry pick now, then if something change I need to this in this PR, it's only Mock data, we can replace it easy. Move mock file to some shared folder please, so we can use it in 2 places